docs/Displaying a User Interface/Template Custom Buttons/index.md

```diff
@@ -98,6 +114,13 @@ List<SoftButtonCapabilities> softButtonCapabilitiesList = sdlManager.getSystemCa
 boolean imageSupported = (!softButtonCapabilitiesList.isEmpty()) ? softButtonCapabilitiesList.get(0).getImageSupported() : false;
 ```
 !@
+
+@![javascript]
+```js
+const softButtonCapabilitiesList = sdlManager.getSystemCapabilityManager().getDefaultMainWindowCapability().getSoftButtonCapabilities();
+const imageSupported = (!softButtonCapabilitiesList.isEmpty()) ? softButtonCapabilitiesList[0].getImageSupported() : false;
```

Comment:
`isEmpty()` not a valid method in JS. Recommend `softButtonCapabilitiesList.length !== 0` for the condition here